src/src.11tydata.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    date: "git Last Modified"
```

Comment:
File needs correct format. 
Run `pnpm|npm run test:eslint` to catch linting errors

Reply:
Fixed. We shiould add this into the automated build :-)